extra/cache-extra/Node/CacheNode.php

```diff
@@ -40,7 +40,7 @@ public function compile(Compiler $compiler): void
             ->addDebugInfo($this)
             ->raw('$this->env->getRuntime(\'Twig\Extra\Cache\CacheRuntime\')->getCache()->get(')
             ->subcompile($this->getNode('key'))
-            ->raw(", function (\Symfony\Contracts\Cache\ItemInterface \$item) use (\$context, \$macros) {\n")
+            ->raw(", function (\Symfony\Contracts\Cache\ItemInterface \$item) use (\$context, \$macros, \$blocks) {\n")
```

Comment:
This bug is independent of the new CaptureNode feature, but it's now more obvious.